### PR TITLE
docs(webapps): add documentation about message correlation

### PR DESCRIPTION
**Pull request by tasso94**
related to CAM-13889

Needs to be squashed before merge.

### CODE REVIEW

File: content/webapps/cockpit/bpmn/correlate-message.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+* **Message Name**\
+  Enter a message name or leave empty. If empty, any execution matches the correlation.
+* **Variables**\
+  Add variables that are set into the root scope of the process instances.
+* **Process Instances**\
+  Select a set of process instances via the checkboxes or by defining query criteria. 
+
+## Process Action
+
+You can find a {{< glyphicon name="envelope" >}} button on the right side next to the BPMN diagram
```
Comment (marstamm) [inline content/webapps/cockpit/bpmn/correlate-message.md:30]: I just noticed how weird the `glyphicon` template is. It creates a semantic button without an attached action. This screws with accessibility, screenreaders and Tab-Order. I created [a ticket](https://jira.camunda.com/browse/CAM-13932) to fix it.

File: content/webapps/cockpit/bpmn/correlate-message.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+{{< img src="../../img/correlate-message/process-action.png" alt="Process action to correlate a message" caption="Process action to correlate a message" >}}
+
+If you click this button, a modal dialog opens:
+
+{{< img src="../../img/correlate-message/modal-dialog.png" alt="Modal dialog to correlate a message" caption="Modal dialog to correlate a message" >}}
+
+When you click into the text input field, you can choose a message name from a list of suggestions.
+
+You can also enter another message name that is not present in the list of suggestions, e.g., it is
+resolved via an EL expression and not present in the list. You can also leave the message name empty.
```
Comment (marstamm) [inline content/webapps/cockpit/bpmn/correlate-message.md:42]: > EL expression

I was not aware of this term. Is this a known term for a cockpit operator? Would it be enough to just call it an expression?

File: content/webapps/cockpit/bpmn/correlate-message.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+To proceed, click on the "Open Batch Operation" button. The correlate message batch operation page
```
Comment (marstamm) [inline content/webapps/cockpit/bpmn/correlate-message.md:44]: We can link to the batch operation from here as well:
```suggestion
To proceed, click on the "Open Batch Operation" button. The correlate message [batch operation]({{< ref "/webapps/cockpit/batch/batch-operation.md" >}}) page
```

File: content/webapps/cockpit/bpmn/correlate-message.md (hunk 1 above)
Comment (tasso94) [reply content/webapps/cockpit/bpmn/correlate-message.md:30]: 👍

File: content/webapps/cockpit/bpmn/correlate-message.md (hunk 2 above)
Comment (tasso94) [reply content/webapps/cockpit/bpmn/correlate-message.md:42]: I agree. It is rather a known term for Java folks.

File: content/webapps/cockpit/bpmn/correlate-message.md (hunk 3 above)
Comment (tasso94) [reply content/webapps/cockpit/bpmn/correlate-message.md:44]: 👍

**Review by marstamm — APPROVED** [review]
:rocket:

**Pull request merged**